app/src/test/java/com/kickstarter/viewmodels/ProjectPageViewModelTest.kt

```diff
@@ -233,71 +248,76 @@ class ProjectPageViewModelTest : KSRobolectricTestCase() {
     @Test
     fun testMediaPlayButton_whenHasVideoFalse_shouldEmitFalse() {
         val project = ProjectFactory.initialProject().toBuilder().video(null).build()
-        val currentUser = MockCurrentUser()
+        val currentUser = MockCurrentUserV2()
         val environment = environment()
             .toBuilder()
-            .currentUser(currentUser)
+            .apolloClientV2(apolloClientSuccessfulGetProject())
+            .currentUserV2(currentUser)
             .build()
 
         requireNotNull(environment.currentConfig()).config(ConfigFactory.config())
```

Comment:
There is a config V2 as well currentConfigV2()
`
val config = ConfigFactory.configForUSUser()
        val currentConfig = MockCurrentConfigV2()
        currentConfig.config(config)
`